src/controller.cc

```diff
@@ -294,6 +294,18 @@ void Controller::PrintEpochStats() {
     return;
 }
 
+void Controller::PrintEpochStats(int tag) {
```

Comment:
What's the advantage of using a tag rather than automatically incrementing the epoch?

Reply:
The tag is used in hammerblade's vanilla stats profiler, so the point of this commit is to align dramsim printstats, with hammerblade. we dont want to have each epoch have its own tag